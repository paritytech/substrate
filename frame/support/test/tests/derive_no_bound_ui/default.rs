trait Config {
	type C;
}

#[derive(frame_support::DefaultNoBound)]
struct Foo<T: Config> {
	c: T::C,
}

fn main() {}
