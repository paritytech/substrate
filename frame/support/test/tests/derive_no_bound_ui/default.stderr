error[E0277]: the trait bound `<T as Config>::C: std::default::Default` is not satisfied
 --> tests/derive_no_bound_ui/default.rs:7:2
  |
7 |     c: T::C,
  |     ^ the trait `std::default::Default` is not implemented for `<T as Config>::C`
